README.md

```diff
@@ -62,13 +62,24 @@ For more configuration options see [here](../../wiki/Configuration).
 
 Need more details? You can find more detailed installation guides for [Unix](../../wiki/Unix) (including your Raspberry Pi), [macOS](../../wiki/macOS), and [Windows](../../wiki/Windows).
 
-#### Building via Docker
-
-*Note*: Using this method, the bot will lose all sounds when restarting because the Docker container has its own file system.
-
-+ Simply clone the repo and run `docker build -t soundbot .` inside the folder.
-+ Afterwards start the bot via `docker run soundbot`.
-+ To run the container in the background use `docker run -d soundbot`.
+#### Building/Running via Docker
+
++ First, clone the repo and run `docker build -t soundbot .` inside the folder.
++ Make a separate directory where your soundbot's data will live (Ex. `/etc/discord-soundbot`)
++ Then create a `docker-compose.yml` file in that directory with the following contents:
+```yaml
+version: '2'                                           
+services:                                              
+  discord-soundbot:                                    
+    image: soundbot
+    volumes:                                           
+    - ./sounds:/app/sounds                             
+    - ./config.json:/app/config/config.json            
+    - ./db.json:/app/db.json                           
```

Comment:
Doesn't work if the user does not have a `db.json` file. Anyway to fix this so that the db is copied over from the container to the user's computer in the end, if he didn't have that file before like with the newly created sounds?

Reply:
So in the README there's already [this part](https://github.com/markokajzer/discord-soundbot#configuration) about the configuration, so it'd say users are already prompted to create a config. It's needed anyway because we need the discord specific token stuff.

Can you just create an empty `db.json` if there is none, use `./config/config.json:/app/config/config.json` for the volume and commit the `docker-compose.yml`. Would that work?